Source/Modules/php.cxx

```diff
@@ -646,16 +665,19 @@ class PHP : public Language {
     Printf(s_wrappers, "/* end wrapper section */\n");
     Printf(s_vdecl, "/* end vdecl subsection */\n");
 
+    if (Len(classes) > 0)
+      Printf(all_cs_entry, " { NULL, NULL, NULL }\n};\n\n");
+
     Dump(f_runtime, f_begin);
     Printv(f_begin, s_header, NIL);
     if (directorsEnabled()) {
       Dump(f_directors, f_begin);
     }
     Printv(f_begin, s_vdecl, s_wrappers, NIL);
-    Printv(f_begin, all_cs_entry, "\n\n", s_arginfo, "\n\n", s_entry,
+    Printv(f_begin, s_arginfo, "\n\n", all_cs_entry, "\n\n", s_entry,
 	" SWIG_ZEND_NAMED_FE(swig_", module, "_alter_newobject,_wrap_swig_", module, "_alter_newobject,NULL)\n"
 	" SWIG_ZEND_NAMED_FE(swig_", module, "_get_newobject,_wrap_swig_", module, "_get_newobject,NULL)\n"
-	" ZEND_FE_END\n};\n\n", NIL);
+	" { NULL, NULL, NULL }\n};\n\n", NIL);
```

Comment:
I'll use ZEND_FE_END. The thing was ZEND_FE_END in module functions, that is non-class. And ZEND_ME_END in class functions I think. I had got some errors using FE_END in class functions. So kept it on NULL. Will change it